pandas/core/groupby/groupby.py

```diff
@@ -2000,23 +2006,52 @@ def std(self, ddof: int = 1):
         ddof : int, default 1
             Degrees of freedom.
 
+        engine : str, default None
+            * ``'cython'`` : Runs the operation through C-extensions from cython.
+            * ``'numba'`` : Runs the operation through JIT compiled code from numba.
+            * ``None`` : Defaults to ``'cython'`` or globally setting
+              ``compute.use_numba``
+
+            .. versionadded:: 1.4.0
+
```

Comment:
in a followup can you try to use shared docstrings here (and the other functions) for the engine/engine_kwargs

Reply:
Sure I can try to upstream these doc parameters from rolling too